about.html

```diff
@@ -0,0 +1,112 @@
+          <li class="top-nav-item icon"><i class="fa-brands fa-facebook"></i></li>
+          <li class="top-nav-item icon"><i class="fa-brands fa-twitter"></i></li>
+          <li class="top-nav-item github">
+            <a
+              href="https://github.com/GutemaG/module-1-capstone.git"
+              target="_blank"
+              >
+              <!-- rel="noopener noreferrer" -->
+              <i class="fa-brands fa-github"></i>
+            </a>
+          </li>
+          <li class="top-nav-item">English</li>
+          <li class="top-nav-item">Logout</li>
+          <li class="top-nav-item">MyPage</li>
+        </ul>
+        <button class="menu" id="nav-menu-show">
+          <i class="fa-solid fa-bars fa-2xl"></i>
+        </button>
+        <a href="./index.html" class="logo">
+          <img src="assets/panafrica2022-logo.png" alt="pan African Logo" >
+        </a>
+        <ul class="nav-items">
+          <li class="nav-close-icon">
+            <i class="fa-solid fa-xmark"></i>
+          </li>
+          <li class="nav-item"><a href="./index.html">Home </a></li>
+          <li class="nav-item active"><a href="./about.html">About </a></li>
+        </ul>
+      </nav>
+      <div class="hero-container">
+        <div class="hero">
+          <p class="hero-text">"Hello! Sharing World"</p>
+          <h1 class="hero-leading-text">
+            Pan African Conference on Artificial Intelligence 2022
+          </h1>
+          <h5 class="hero-description">
+            A key topic in digitization and industry 4.0. in line with global
+            trends, the Middle East and Africa expect a 400% turnover increased
+            by 2025. AI is a UNESCO development priority in 21 of 32 African
+            countries surveyed.
+          </h5>
+          <h3 class="hero-date">2022.10.4(THU) ~ 5(FRI)</h3>
+          <span>@Ethiopia Science Museum, Addis Ababa Ethiopia</span>
+        </div>
+      </div>
+    </header>
+    <main class="main">
+      <section class="about-main-top">
+        <h3 class="heading-text title">
+          Pan African Conference on Artificial Intelligence 2022
+        </h3>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nemo, culpa.
+        </p>
+        <div class="about-main-logo">
+          <img src="assets/panafrica2022-logo.png" alt="PanAfrica logo" >
+        </div>
+      </section>
+      <section class="past-summit-container">
+        <h3 class="heading-text title">See the past PanAfrican Conferences</h3>
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Debitis,
+          natus.
+        </p>
+        <ul class="past-summits">
+          <li class="past-summit">
+            <img src="assets/topics/AI-Agri.png" alt="AI in Agriculture" >
+            <div class="overlay"></div>
+            <div class="past-summit-content">
+              <h3>2022</h3>
+              <p>AI in Agriculture</p>
+            </div>
```

Comment:
- **About: Mobile size**: 
The pages should look almost identical to the original design. Here you left behind the partner section
- Original design:
![2](https://user-images.githubusercontent.com/63972670/203525760-91b5ba4f-31e1-4661-9552-00bf1c358484.PNG)
- Your Design:

![Capture1](https://user-images.githubusercontent.com/63972670/203525974-5b400566-224e-4915-8ff9-921cced4473f.PNG)
NB: Open this Link to see the 👉  [Original Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)